docs/src/assets/0_pregenerate.jl

```diff
@@ -16,3 +17,61 @@ let
              xlabel="Ecut", ylabel="energy absolute error", label="")
     savefig(p, "convergence_study_ecut.png")
 end
+
+let
+    include("../../../../examples/pseudopotentials.jl")
+
+    function run_scf(Ecut, psp)
+        println("Ecut = $Ecut")
+        println("----------------------------------------------------")
+        a = 5.0
+        lattice   = a * Matrix(I, 3, 3)
+        atoms     = [ElementPsp(:Li, psp=psp)]
+        positions = [zeros(3)]
+    
+        model = model_LDA(lattice, atoms, positions; temperature=1e-2)
+        basis = PlaneWaveBasis(model; Ecut=Ecut, kgrid=[8, 8, 8])
+        self_consistent_field(basis; tol=1e-12)
+    end
+    
+    function converge_Ecut(Ecuts, psp, tol)
+        energy_ref = run_scf(Ecuts[end], psp).energies.total
+        energies = []
+        for Ecut in Ecuts
+            energy = run_scf(Ecut, psp).energies.total
+            push!(energies, energy)
+            if abs(energy - energy_ref) < tol
+                break
+            end
+        end
+        n_energies = length(energies)
+        errors = abs.(energies .- energy_ref)
+        iconv = findfirst(errors .< tol)
+        (Ecuts=Ecuts[begin:n_energies], errors=errors,
+         Ecut_conv=Ecuts[iconv], error_conv=errors[iconv])
+    end
+
+    Ecuts = 20:4:140
+    tol   = 3.675e-5 / 1  # 1.0 meV / atom
```

Comment:
Might as well define `n_atom = 1` and use `austrip(1e-3u"eV")` to make this clear.